js/src/components/modals/SemRushLoading.js

```diff
@@ -0,0 +1,31 @@
+/* External dependencies */
+import React from "react";
+import { __, sprintf } from "@wordpress/i18n";
+import { Fragment } from "@wordpress/element";
+import { SvgIcon } from "@yoast/components";
+
+/**
+ * Creates the loading content for the SEMRush related keywords modal.
+ *
+ * @returns {React.Element} The SEMRush loading content.
+ */
+const SemRushLoading = () => {
+	return (
+		<Fragment>
```

Comment:
Not sure we need a Fragment, as all the content is already wrapped in a paragraph. Seem unnecessary to me and I'd consider to remove it, unless I'm missing something.

Reply:
This content is based on the `SemRushLimitReached` content. So if this is deemed unnecessary, we should come to an agreement on how to structure these components.

Reply:
The `SemRushLimitReached` content is made of a paragraph and a sibling `UpdateSemRushPlanLink` so these 2 need a wrapper component. That's why a Fragment is necessary there. The `SemRushLoading` doesn't need a Fragment instead.